crates/astria-sequencer/src/app.rs

```diff
@@ -293,13 +297,35 @@ impl App {
                 continue;
             };
 
-            // store transaction execution result, indexed by tx hash
             let tx_hash = Sha256::digest(&tx);
+
+            // Sum up the total bytes in sequence actions
+            let mut sequenced_data_size = 0;
+            for action in signed_tx.clone().into_unsigned().actions {
+                if let Some(sequence_action) = action.as_sequence() {
+                    sequenced_data_size += sequence_action.data.len();
+                }
+            }
+
+            // Don't include tx if it would make the sequenced block data too large.
+            if total_sequenced_data_size + sequenced_data_size > MAX_BLOCK_SEQUENCE_DATA_SIZE {
+                debug!(
```

Comment:
I wonder if this should be flagged more prominently. Maybe an `info!` level that informs you of how many txes were dropped?

Reply:
This is probably a place we want metrics, I can add a counter of some sort and log for now.

This was done as a debug to give as much information as should be needed about why tx not included.